docs/TestRunner.md

```diff
@@ -0,0 +1,131 @@
+# Test Parser
+
+## Overview 
+
+Parse a [yml file](https://docs.google.com/document/d/17GOv1yVAKY4vmOd1Vhg_IitpyCMiX-e_b09eufNysYI/edit#heading=h.9dwsxj6quakr) into a test object.
+
+## How to use it
+
+Create a TestParser using the path of the yml file.
+
+```
+const TestParser = require('skill-testing-ml').TestParser;
+const parser = new TestParser("en-US/edgar.yml");
+
+```
+
+Call the parse method, if the file is valid an object will be created, if not an exception will be through with the error.
+
+```
+parser.parse();
+```
+
+Error example:
+```
+Error - YAMLException:
+end of the stream or a document separator is expected at line 17, column 14:
```

Comment:
Please note here that the error object will also have a "line" property that will have the line number on it